protocol_rfcs/variant-type.md

```diff
@@ -68,9 +67,11 @@ When Variant type is supported (`writerFeatures` field of a table's `protocol` a
 ## Reader Requirements for Variant Data Type
 
 When Variant type is supported (`readerFeatures` field of a table's `protocol` action contains `variantType`), readers:
-- must be able to read the two parquet struct fields, `value` and `metadata` and interpret them as a Variant in concordance with the [Variant binary encoding specification](https://github.com/apache/spark/blob/master/common/variant/README.md).
-- It is recommended but not required for a Delta reader to treat the struct as a single indivisible Variant field, if the reader is used in an engine or other context that supports Variant.
-- can ignore any parquet struct field names starting with `_` (underscore)
+- must recognize and tolerate a `variant` data type in a Delta schema
+- must use the correct physical schema (struct-of-binary, with fields `value` and `metadata`) when reading a Variant data type from file
+- must make the column available to the engine:
+    - [Recommended] Expose and interpret the struct-of-binary as a single Variant field in accordance with the [Spark Variant binary encoding specification](https://github.com/apache/spark/blob/master/common/variant/README.md).
+    - [Alternate] Expose the raw physical struct-of-binary, e.g. if the engine does not support Variant.
```

Comment:
Is it also an acceptable alternate, to expose a string column by internally converting the struct-of-binary? Gives up all the benefits of the variant encoding, but maximizes compat with engines that don't allow users to load the library code that would interpret the struct-of-binary directly?

Reply:
Yeah, that should work. I added that as another alternate.